lib/ansible/modules/cloud/ovirt/ovirt_nic.py

```diff
@@ -255,7 +267,12 @@ def main():
             for vnic in connection.system_service().vnic_profiles_service().list():
                 if vnic.name == profile and vnic.network.id == network.id:
                     entitynics_module.vnic_id = vnic.id
-
+        else:
+            # When not specified which vnic use ovirtmgmt/ovirtmgmt
+            networks = networks_service.list()
+            vnics = get_vnics(networks_service, networks, connection)
+            if len(networks) == 1 and len(vnics) == 1:
+                entitynics_module.vnic_id = vnics[0].id
```

Comment:
This might be wrong as you can change the vnic accidentally if there is empty network set.